README.md

```diff
@@ -50,27 +50,23 @@ NB: multiple CouchDB nodes will be more complicated, but the general pattern out
 
 By default CouchDB runs in "admin party" mode, which means you do not need users to read or edit any data. This is great for some, but to use Medic safely we're going to disable this feature.
 
-First, add an admin user:
+First, add an admin user. When prompted to create an admin during installation, use username:`admin` and password:`pass`. Passwords can be changed via [Fauxton](http://localhost:5984/_utils). For more information see the [CouchDB install doc](http://docs.couchdb.org/en/2.0.0/install/).
 
-- If you are running CouchDB 2.x there is a wizard in Fauxton that does this for you, as well as initialising some databases. For more information see the [CouchDB install doc](http://docs.couchdb.org/en/2.0.0/install/).
-- If you are running CouchDB 1.x, run the following command to add an admin user:
+Now that's done, we must reconfigure CouchDB to require authentication. To be able to use Fauxton with authenticated users:
```

Comment:
I'd rather we sort these requests in order of importance: first the `require_valid_user`, then user creation and then changing WWW-Authenticate realm.

Reply:
totally